mangaki/mangaki/management/commands/tokens.py

```diff
@@ -1,8 +1,16 @@
 from django.core.management.base import BaseCommand
+from django.core.mail import send_mail
 from django.conf import settings
 from mangaki.utils.tokens import compute_token
 from django.contrib.auth import get_user_model
 from django.db import connection
+from django.conf import settings
+import os
+import yaml
+from jinja2 import Template
+
+
+DEBUG_USERNAMES = ['jj', 'RaitoBezarius']
```

Comment:
Why do we have this `DEBUG_USERNAMES` dance instead of doing `python manage.py tokens jj RaitoBezarius`?

(BTW this should be called `python manage.py newsletter` or something, no?)